IdentityCore/src/cache/accessor/MSIDAccountCredentialCache.h

```diff
@@ -163,15 +163,30 @@
 - (BOOL)saveWipeInfoWithContext:(nullable id<MSIDRequestContext>)context
                           error:(NSError * _Nullable * _Nullable)error;
 
+/*
+ Saves app metadata
+ */
 - (BOOL)saveAppMetadata:(nonnull MSIDAppMetadataCacheItem *)metadata
                 context:(nullable id<MSIDRequestContext>)context
                   error:(NSError * _Nullable * _Nullable)error;
 
 /*
- Gets app metadata for a particular key
+ Gets app metadata for a particular query
+ */
+- (nullable MSIDAppMetadataCacheItem *)getAppMetadataWithQuery:(nonnull MSIDAppMetadataCacheQuery *)query
```

Comment:
should this return an array as a result? (it is a query afterall).
Also, no need for "get" here.

Reply:
Agree. Updated the code and fixed the issue .